packages/playwright-ct-react/hooks.d.ts

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-import type { JsonObject } from '@playwright/experimental-ct-core/types/component';
+export interface RegisterHooksConfig {}
```

Comment:
NOTE: `RegisterHooksConfig` is specified for [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html). See https://github.com/microsoft/playwright/issues/19933 for more info